book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -1,5 +1,5 @@
+(cl-inclusive-events-planning)=
 # Planning an Inclusive Event
-
 Running an event that is inclusive of all is more than just a choice; it’s a necessity to ensure that all your participants will be able to fully engage in the event. 
```

Comment:
Running an event that is inclusive of all is more than just a choice; it’s a necessity to guarantee that all your participants will be able to fully engage in the event. 
```suggestion
Running an event that is inclusive of all is more than just a choice; it’s a necessity to ensure that all your participants will be able to fully engage in the event. 
```